recipe/build.sh

```diff
@@ -85,10 +80,7 @@ elif [[ "${target_platform}" == linux-* ]]; then
   sed -i "s/# define V8_HAS_CPP_ATTRIBUTE_NODISCARD (V8_HAS_CPP_ATTRIBUTE(nodiscard))//g" include/v8config.h
 fi
 
-find out.gn -type f -name '*.ninja' -exec sed -i "
-    s|-Werror||g
-    s|^\s*ldflags = |&${LDFLAGS} |
```

Comment:
I just removed this again since it made the macOS builds fail and I won't debug those at his time.